doc/sphinx/source/community/detailed_release_procedure.rst

```diff
@@ -0,0 +1,190 @@
+.. _detailed-release-procedure:
+
+Release: recipes runs and comparison
+====================================
+
+The release procedure for ESMValTool is a fairly involved process (at the moment), so it
+is important to be very well organized and to have documented each procedural steps, so that
+the next release manager can follow said steps, and finalize the release without any delays.
+
+The workflow below assumes an ESMValCore release candidate, or a completed stable release, have been released
+and deployed on conda-forge and PyPI; it also assumes the release manager has access to accounts on DKRZ/Levante.
+
+Below is a list of steps that the release manager, together with the previous release manager, should go through before the actual release;
+these include testing the new code by running all available recipes in yhe `main` branch, and comparing the output against
+the previous release.
+
+Open an issue on GitHub
+-----------------------
+
+First, open an issue on GitHub where the testing workflow before the release is documented (see example https://github.com/ESMValGroup/ESMValTool/issues/2881).
+Name it something relevant like "Recipe testing and comparison for release 2.x.x", and populate the issue description with information
+about where the testing is taking place, what tools are used, and what versions, here are some suggestions:
+
+
+Where the work is done e.g. work done on DKRZ/Levante:
+
+- submit files in `/home/b/b382109/submit` (for v2.7.0)
+- output in `/scratch/b/b382109/esmvaltool_output` (for v2.7.0)
+
+We should document various utilities' versions so that the work can be reproduced in case there
+is an issue, or release work needs to be picked up mid-release by another release manager:
+
+- documenting `conda`/`mamba` versions:
+
+.. code-block:: bash
+
+  mamba --version
+
+- documenting `git` branch and its state:
+
+.. code-block:: bash
+
+  git status
+
+Furthermore, the runtime environment needs to be documented:
+
+.. code-block:: bash
+
+  mamba env create -n tool2xxTest -f environment.yml
+  conda activate tool2xxTest
+
+Make a copy of the environment file, and attach it in the release testing issue; to
+record the environment in a yaml file use e.g. `conda env export > ToolEnv270Test.txt`.
+
+Mention any special data movement one needs to perfom to run the tests e.g.:
+for v2.7.0 the release manager moved the autoassess-specific files to
+`/home/b/$USER/autoassess_files` on DKRZ/Levante.
+
+Modifications to configuration files need to be documented as well: for example,
+for v2.7.0 the release manager had to modify `config-user.yml` file by
+adding a number of extra paths for DKRZ/Levante-specific data pools:
+
+.. code-block:: yaml
+
+  CMIP6:
+    - /work/bd0854/DATA/ESMValTool2/CMIP6_DKRZ
+    - /work/bd0854/DATA/ESMValTool2/download/CMIP6
+  CMIP5:
+    - /work/bd0854/DATA/ESMValTool2/CMIP5_DKRZ
+    - /work/bd0854/b309141/additional_CMIP5
```

Comment:
Does this directory contain retracted data that is no longer available on ESGF? It is against our [broken recipe policy](https://esmvaltool--3032.org.readthedocs.build/en/3032/community/broken_recipe_policy.html) to include recipes using such data in a release. Therefore I would recommend against using anything but the main data pool from the cluster (which will be hopefully cleaned up by the cluster admins) and some recently downloaded data from ESGF.

Reply:
At DKRZ, the problem is not so much with CMIP6 data but rather with the CMIP5 pool that is not really maintained and from which lots of data have been removed last year to free disk space. So I'd like to continue using the `/work/bd0854/DATA/ESMValTool2/download/CMIPX`, otherwise results in lots of downloads for every testing round.

Reply:
indeed - relates to the comment above about publicly available data

Reply:
I can understand that, though some occasional cleanup might be good. But what about `/work/bd0854/b309141/additional_CMIP5`? It looks like someone's home directory?